apps/router/src/hooks/guardian/useGuardianSetup.tsx

```
@@ -0,0 +1,239 @@
+          type: SETUP_ACTION_TYPE.SET_PROGRESS,
+          payload: SetupProgress.ConnectGuardians,
+        });
+        break;
+      case GuardianServerStatus.VerifyingConfigs:
+      case GuardianServerStatus.VerifiedConfigs:
+        dispatch({
+          type: SETUP_ACTION_TYPE.SET_PROGRESS,
+          payload: SetupProgress.VerifyGuardians,
+        });
+        break;
+      case GuardianServerStatus.ConsensusRunning:
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)